### ISSUE TITLE
Multi-source application is making argocd UI to lose information

### ISSUE THREAD

**Issue by amedinagar** [argoproj/argo-cd#13215]
# Summary
We are making use of an application CRD that calls an applicationset defined in a helm chart, to get the values, we use the new feature of [multi source](https://argo-cd.readthedocs.io/en/stable/user-guide/multiple_sources/) .

# Motivation

The problem comes from the fact that in the main view of the application wrapper, the information of the last commit does not appear, besides that the information of the last sync does not reflect the changes in the child applications.

the ui has a warning:
![argo_multiple](https://user-images.githubusercontent.com/82578270/231741898-41c12b2c-0697-43f9-9351-01da2c228471.png)

I understand that this problem has already been foreseen.
In our case we would like this information not to be lost, so that the gitops flow is clearer from the main view.

# Proposal

```
spec:
  project: <project>
  sources:
  - repoURL: <url>
    targetRevision: HEAD
    ref: <repo>
  - repoURL: <chart_url>
    targetRevision: <chart>
    chart: applicationset
    helm:
      valueFiles:
        - $<repo>/<route>
```
it occurs to me that in this case, we could choose one of the two sources for the system to track the info, for example choose the git repo, and track the changes only from the chosen source, I do not know how complicated this change can be, but it would be quite useful in our case.
Thank you very much!

**Comment by JJotah** [comment]
+1

**Comment by sergusha** [comment]
+1

**Comment by AndreCbrera** [comment]
+1

**Comment by chenfli** [comment]
+1

**Comment by gabsdan** [comment]
+1

**Comment by sambonbonne** [comment]
I hate being the one to say it but maybe the *+1* are not needed, AFAIK ArgoCD developers want to solve this issue to get multi-source applications stable for everyone so they don't need notifications to remind them to work.

**Comment by amedinagar** [comment]
I guess this is resolved in 2.12, so i will close the issue.
